portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,35 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    private final ArrayList<String> messages = new ArrayList<String>();
 
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    messages.add("Bad Bunny");
+    messages.add("SZA");
+    messages.add("Rico Nasty");
+    messages.add("Frank Ocean");
+    messages.add("Joyce Manor");
```

Comment:
Can be initialized as List<String> messages = new ArrayList<>(Arrays.asList("Bad Bunny", "SZA", ...)) instead of doing this for every function call.